services/subscription.go

```diff
@@ -28,7 +28,8 @@ const (
 // RunLogTopic is the signature for the Request(uint256,bytes32,string) event
 // which Chainlink RunLog initiators watch for.
 // See https://github.com/smartcontractkit/chainlink/blob/master/solidity/contracts/Oracle.sol
-var RunLogTopic = common.HexToHash("0xd27ce9cd40e3b9de8d013e1c32693550a6f543fec0191156dc826978fffb3f48")
+// If updating this, be sure to update the truffle suite's "expected event signature" test.
```

Comment:
Fair enough, but cutting the other test makes me itch for a real RunLog integration test.

Reply:
👍 We can make that part of the examples/echo_server/runlog tests story that's already coming. It would be easy enough to add to here:

https://github.com/smartcontractkit/chainlink/blob/master/internal/ci/truffle_test#L8